### PR TITLE
Debug: enable pprof

### PR DESCRIPTION
**Pull request by d4rk5h0t**
## Description
enable pprof server for zarb by starting server to gather profiling data
its not recommended to open leave this one open to the internet
```bash
zarb start -w ./ -d 6060
```

## Related issue(s)
---

## Checklist

- [x] New feature (non-breaking change which adds functionality)
- [x] My code follows the code style of this project.
- [x] All new and existing tests passed.
- [x] CHANGELOG is updated.

### CODE REVIEW

File: cmd/zarb/start.go [hunk 1]
```diff
@@ -38,6 +40,10 @@ func Start() func(c *cli.Cmd) {
 			Name: "a auth",
 			Desc: "Passphrase of the key file",
 		})
+		pprofOpt := c.String(cli.StringOpt{
```
Comment (b00f) [inline cmd/zarb/start.go:43]: Why not boolean. Default set to false

File: cmd/zarb/start.go [hunk 2]
```diff
@@ -84,6 +90,19 @@ func Start() func(c *cli.Cmd) {
 				return
 			}
 
+			// separate pprof handlers from DefaultServeMux.
+			pprofMux := http.DefaultServeMux
+			http.DefaultServeMux = http.NewServeMux()
+			if pprofOpt != nil && *pprofOpt != "" {
```
Comment (b00f) [inline cmd/zarb/start.go:96]: `pprofOpt != nil` As I checked `mow.cli` option will never be null.

File: cmd/zarb/start.go (hunk 1 above)
Comment (d4rk5h0t) [reply cmd/zarb/start.go:43]: it's not recommended to Expose this server Online so its better to let user expose it on any interface he/she likes by providing that interface specific address

by default if this address not supplied server will not expose any data to outside

File: cmd/zarb/start.go (hunk 2 above)
Comment (d4rk5h0t) [reply cmd/zarb/start.go:96]: @b00f  in that case it's redundant

File: cmd/zarb/start.go [hunk 3]
```diff
@@ -38,6 +40,10 @@ func Start() func(c *cli.Cmd) {
 			Name: "a auth",
 			Desc: "Passphrase of the key file",
 		})
+		pprofOpt := c.String(cli.StringOpt{
+			Name: "d pprof",
+			Desc: "debug pprof server address Address(not recommended to expose to internet)",
```
Comment (b00f) [inline cmd/zarb/start.go:45]: I suggest this:
```
Name: "pprof",			
Desc: "debug pprof server address (not recommended to expose to internet)",
```

- Remove one letter option
- Address address -> address

Comment (d4rk5h0t) [reply cmd/zarb/start.go]: applyed

**Review by b00f — APPROVED** [review]

**Pull request merged**